File: 1012.py
A,B,C=map (float,input().split(" "))
areaT = ((A*C)/2)
areaC = (3.14159*(C**2))
areaTR= ((A+B)/2)*C
areaQ= B**2
areaR=A*B
print('TRIANGULO: {:.3f}'.format(areaT))
print('CIRCULO: {:.3f}'.format(areaC))
print('TRAPEZIO: {:.3f}'.format(areaTR))
print('QUADRADO: {:.3f}'.format(areaQ))
print('RETANGULO: {:.3f}'.format(areaR))

File: 1012.c
#include <stdio.h>
#include <math.h>
int main(){
    float a,b,c;
    scanf ("%f %f %f",&a,&b,&c);
    float triangulo;
    triangulo = ((a*c)/2);
    float circulo;
    circulo =  3.14159*pow(c,2);
    float trapezio;
    trapezio = ((a+b)*c)/2;
    float quadrado;
    quadrado = b*b;
    float ret;
    ret = a*b;
    printf("TRIANGULO: %.3f\n",triangulo);
    printf("CIRCULO: %.3f\n",circulo);
    printf("TRAPEZIO: %.3f\n",trapezio);
    printf("QUADRADO: %.3f\n",quadrado);
    printf("RETANGULO: %.3f\n",ret);
}   
